cli/src/host.rs

```diff
@@ -284,3 +290,31 @@ fn host_to_windows_path(path: &Path) -> String {
         .trim_start_matches("\\\\?\\")
         .into_owned()
 }
+
+// From https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
```

Comment:
Can you describe why this is copied?  Feels wrong (also from a licensing standpoint)

Reply:
It's a routine to normalize `..` and `.` in paths. 

> also from a licensing standpoint

You think? Both are Apache 2.0.

Reply:
Removed, I'm now using [typed-path](https://github.com/chipsenkbeil/typed-path) which provides a `normalized` method.